src/plugins/data_views/docs/openapi/components/schemas/create_data_view_request_object.yaml

```diff
@@ -0,0 +1,44 @@
+title: Create data view request
+type: object
+required:
+  - data_view
+properties:
+  data_view:
+    type: object
+    required:
+      - title
+    description: The data view object.
+    properties:
+      allowNoIndex:
+        $ref: 'allownoindex.yaml'
+      fieldAttrs:
+        $ref: 'fieldattrs.yaml'
+      fieldFormats:
+        $ref: 'fieldformats.yaml'
+      fields:
+        type: object
```

Comment:
seems like this should be defined

Reply:
Yup, those objects definitely will require more details in the long-run, but the intent of this initial PR is to just get the shape of the APIs covered and be at least as detailed as the docs, with further additions to follow as necessary.